src/services/LocalStorageService.ts

```diff
@@ -1,3 +1,11 @@
+import { LOCAL_STORAGE_KEYS } from 'constants/general';
+
+interface Headers {
+  client?: string;
+  token?: string;
+  uid?: string;
+}
```

Comment:
this interface should be used in `setHeaders` function, so i would move this to general `utils/types`